packages/desktop-client/src/components/transactions/TransactionsTable.js

```diff
@@ -732,47 +735,76 @@ const Transaction = memo(function Transaction(props) {
     setPrevShowZero(showZeroInDeposit);
   }
 
-  function onUpdate(name, value) {
-    if (transaction[name] !== value) {
-      let newTransaction = { ...transaction, [name]: value };
+  var reconciledWarningShowing = false;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏
```suggestion
  let [showReconciliationWarning, setShowReconciliationWarning] = useState(false);
```